docs/dunst.pod

```diff
@@ -264,6 +264,22 @@ The progress value can be set with a hint, too.
 
 =back
 
+=head1 ACTIONS
+
+Dunst allows notifiers (eg: programs that fire the notifications) to specify
```

Comment:
You probably meant `i.e.` here instead of `eg` (`i.e.` can be used as `that is` , `in other words` and `eg` signifies an example) and please add the dots just to please the pendants :)

Also, `fire` may not be the best word to use here, in documentation being literal is usually the best so `send` should probably be used.